pkg/management/component.go

```diff
@@ -534,7 +534,8 @@ func (c *component) ResetPassword(ctx context.Context, realmName string, userID
 	credKc.Type = &passwordType
 
 	if password.Value == nil {
-		// no password value was provided; a new password, that respects the password policy of the realm, will be generated
+		// the commented code respects the following scenario: if the realm has a pwd policy, the generated pwd respects this policy
+		/*// no password value was provided; a new password, that respects the password policy of the realm, will be generated
```

Comment:
/*// is a little bit weird...
It might be more readable with a new line after /*

Reply:
DONE